<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamMediaServer</class>
 <widget class="QWidget" name="StreamMediaServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="PlayerGroupBox">
       <property name="title">
        <string>Preview</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../IDE.qrc">:/Page/rc/Page/VideoScreen.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnStart">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/build.png</normaloff>:/Menu/rc/Menu/build.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnPause">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/pause.png</normaloff>:/Menu/rc/Menu/pause.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnStop">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/stop.png</normaloff>:/Menu/rc/Menu/stop.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="PlayEnable">
            <property name="text">
             <string>Enable</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="MediaQueneGroupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Media Quene</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="MediaQueneList"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnStart_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/previous.png</normaloff>:/Menu/rc/Menu/previous.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnStart_2">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Page/rc/Page/rc.png</normaloff>:/Page/rc/Page/rc.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnPause_3">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/next.png</normaloff>:/Menu/rc/Menu/next.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnPause_4">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/refresh.png</normaloff>:/Menu/rc/Menu/refresh.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="MeidaListGroupBox">
       <property name="title">
        <string>Meida Source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QComboBox" name="MediaType">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>All</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/media.png</normaloff>:/Page/rc/Page/media.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Camera</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Menu/rc/Menu/eye.png</normaloff>:/Menu/rc/Menu/eye.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Video</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/mp4.png</normaloff>:/Page/rc/Page/mp4.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Audio</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/mp3.png</normaloff>:/Page/rc/Page/mp3.png</iconset>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Image</string>
             </property>
             <property name="icon">
              <iconset resource="../IDE.qrc">
               <normaloff>:/Page/rc/Page/jpg.png</normaloff>:/Page/rc/Page/jpg.png</iconset>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="CompleterLineEdit" name="MediaPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="MediaPathSelect">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="MediaTreeList">
          <column>
           <property name="text">
            <string notr="true">Media Source</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Menu/rc/Menu/StreamMedia.png</normaloff>:/Menu/rc/Menu/StreamMedia.png</iconset>
           </property>
          </column>
          <item>
           <property name="text">
            <string>LocalDisk</string>
           </property>
           <property name="icon">
            <iconset resource="../IDE.qrc">
             <normaloff>:/Page/rc/Page/rc.png</normaloff>:/Page/rc/Page/rc.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="UserListGroupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>User List</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="UserTreeList">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="BtnStart_4">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/chat.png</normaloff>:/Menu/rc/Menu/chat.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnStart_5">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/plus.png</normaloff>:/Menu/rc/Menu/plus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="BtnPause_5">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/minus.png</normaloff>:/Menu/rc/Menu/minus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="BtnPause_6">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../IDE.qrc">
              <normaloff>:/Menu/rc/Menu/refresh.png</normaloff>:/Menu/rc/Menu/refresh.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="BtnOpenServer">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OpenServer</string>
       </property>
       <property name="icon">
        <iconset resource="../IDE.qrc">
         <normaloff>:/Menu/rc/Menu/notpass.png</normaloff>:/Menu/rc/Menu/notpass.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CompleterLineEdit" name="ServerConfEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="ServerConfSelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CompleterLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">completertool.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../IDE.qrc"/>
 </resources>
 <connections/>
</ui>
